src/pwm.rs

```diff
@@ -0,0 +1,186 @@
+use crate::println;
```

Comment:
That certainly would be nice. Do you think we should suggest this and then build it into this PR or save it for later? Not sure how long it takes to make changes to the peripheral driver Interface since that would affect all boards?

Reply:
Changed this to `panic` in https://github.com/RIOT-OS/rust-riot-wrappers/pull/38/commits/8d770fd642f3b92c3b94a321de643fe8929d30ff. Hopefully it can be resolved but until that we have the panic.

Reply:
I wouldn't wait for RIOT changes, the panics are fine now.

If it's something that can't easily be done for many boards, it may be that there'll be a peripheral feature this would be limited to ... or maybe it turns out it's just something impractical anyway and the embedded-hal API does something bad, don't know :-)

Reply:
(Leaving this thread as unresolved not because it'll need addressing here, but because there's a potential follow-up issue to be created later on).